scripts/coords.js

```diff
@@ -0,0 +1,70 @@
+
+var _ = require('./constants.js');
+
+
+
+
+// -----------------------------+ private functions +--------------------------------
+
+function LatLon(lat, lon){
+	this.lat = lat;
+	this.lon = lon;
+}
+
+function toRad(n) {
+	return n * Math.PI / 180;
+};
+function toDeg(n) {
+	return n * 180 / Math.PI;
+};
+
+function segment(startLatLon, endLatLon) {
```

Comment:
This doesn't look like it's used anywhere. It also looks like the intent is for this to be `new`'d up, but it starts with a lowercase character. If it were me, I would just use object literals rather than newing up an object.